src/App.jsx

```diff
@@ -1,10 +1,27 @@
 import { MovieListPage } from './Containers/MovieListPage/MovieListPage';
-import styles from './App.module.css';
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+import { fetchMovie } from './requests/requests';
+import { SearchFormContainer } from './Containers/SearchFormContainer/SearchFormContainer';
+import { MovieDetailsContainer } from './Containers/MovieDetailsContainer/MovieDetailsContainer';
+
+const router = createBrowserRouter([
+  {
+    path: '/',
+    element: <MovieListPage />,
+    children: [
+      {
+        path: '/',
+        element: <SearchFormContainer />,
+      },
+      {
+        path: '/:movieId',
+        element: <MovieDetailsContainer />,
+        loader: fetchMovie,
```

Comment:
To decouple `fetchMovie` requester from router interface, it might be better to do it like this:

```suggestion
        loader: ({params}) => fetchMovie(params.movieId),
```

`fetchMovie` should be changes correspondingly to accept only `movieId` `string` argument.